scl/windowspatternparser/windows.conf

```diff
@@ -0,0 +1,23 @@
+
+block parser windows-eventlog-parser() {
+    channel {
+        filter {
+            match("(4624|4634)" value(".SDATA.win@18372\.4.EVENT_ID"));
+        };
+        parser {
```

Comment:
do we need the filter here? db-parser() should be good enough for classification whether the event was identified, so later we can filter on tags, which should be faster.

For this use-case, it might actually make sense to drop unmatched messages in db-parser(), hmmm. sending a pull request in a minute.